styles/styles.css

```diff
@@ -9,20 +9,21 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
 */
-  
+
 /* fallback font for Gellix (normal - 400) */
-  @font-face {
-      font-family: "gellix-normal-400-fallback";
-      size-adjust: 100.871%;
-      src: local("Arial");
-  }
 
-  /* fallback font for Gellix (normal - 700) */
-  @font-face {
+@font-face {
```

Comment:
Not 100% sure but I think we should have 1 `gellix-fallback` `font-family` and two `@font-faces` with difference `size-adjust` and `font-weight`. Otherwise we would have to explicitly set either `gellix-normal-700-fallback` or `gellix-normal-400-fallback` where we make use of it and not just `font-weight`.